docs/streams/quickstart.html

```diff
@@ -140,25 +130,36 @@
     --topic streams-file-input
 </pre>
 
+The created topic can be described with the same <b>kafka-topics</b>> tool:
 
 <pre class="brush: bash;">
-&gt; bin/kafka-console-producer.sh --broker-list localhost:9092 --topic streams-file-input < file-input.txt
+&gt; bin/kafka-topics.sh --zookeeper localhost:2181 --describe
+
+Topic:streams-file-input	PartitionCount:1	ReplicationFactor:1	Configs:
+    Topic: streams-file-input	Partition: 0	Leader: 0	Replicas: 0	Isr: 0
+</pre>
+
+And start the console producer that can then write to this topic line-by-line,
```

Comment:
Also: We should tell users whether they should a. stop the console producer at this point or b. keep it running (to enter another line of text later on).  If we want to go with option (b), then we must also tell the user to start the console producer in a separate terminal.